number.go

```diff
@@ -273,6 +273,126 @@ func (n *Number) NotInDelta(value, delta float64) *Number {
 	return n
 }
 
+// InDeltaRelative succeeds if two numerals are within a relative delta of each other.
+//
+// The relative delta is expressed as a percentage (i.e. 0.01 is 1%).
```

Comment:
nit:

* I wouldn't say it's percentage - user should pass `0.01`, not `1`
* In comment, let's also provide the actual formula we use: `Abs(number-value) / Abs(number) < delta`
* In comment, let's mention that number can't be zero
* In comment, let's mention that number, value, and delta can't be NaN